content/kubermatic/v2.17/tutorials_howtos/KKP_autoscaler/_index.en.md

```diff
@@ -0,0 +1,238 @@
+
+coredns-666448b887-s8wv8              	       1/1     	      Running   	0           36m
+
+coredns-666448b887-vldzz              	       1/1     	      Running  		0           36m
+```
+
+As shown above, the Autoscaler has been provisioned and running. 
+
+
+## Annotating MachineDeployments for Autoscaling
+
+The Cluster Autoscaler only considers MachineDeployment with valid annotations. The annotations are used to control the minimum and the maximum number of replicas per MachineDeployment. You don't need to apply those annotations to all MachineDeployment objects, but only on MachineDeployments that Cluster Autoscaler should consider.
+
+```bash
+cluster.k8s.io/cluster-api-autoscaler-node-group-min-size - the minimum number of replicas (must be greater than zero)
+
+cluster.k8s.io/cluster-api-autoscaler-node-group-max-size - the maximum number of replicas
+```
+
+You can apply the annotations to MachineDeployments once the Cluster is provisioned and the MachineDeployments are created and running by following the steps below. 
+
+**Step 1**
+
+Run the following kubectl command to check the available MachineDeployments:
+
+```bash
+$ kubectl get machinedeployments -n kube-system 
+
+NAME                        AGE  DELETED REPLICAS AVAILABLEREPLICAS PROVIDER  OS   VERSION
+test-cluster-worker-v5drmq 3h56m            2             2           aws    ubuntu 1.19.9 
+test-cluster-worker-pndqd  3h59m            1             1           aws    ubuntu 1.19.9
+```
+
+**Step 2**
+
+The annotation command will be used with one of the MachineDeployments above to annotate the desired MachineDeployments.  In this case, the  `test-cluster-worker-v5drmq` will be annotated, and the minimum and maximum will be set.
+
+### Minimum annotation:
+
+```bash
+kubectl annotate machinedeployment -n kube-system test-cluster-worker-v5drmq cluster.k8s.io/cluster-api-autoscaler-node-group-min-size="1"
+
+machinedeployment.cluster.k8s.io/test-cluster-worker-v5drmq annotated
+```
+
+### Maximum annotation:
+
+```bash
+kubectl annotate machinedeployment -n kube-system test-cluster-worker-v5drmq cluster.k8s.io/cluster-api-autoscaler-node-group-max-size="5"
+
+machinedeployment.cluster.k8s.io/test-cluster-worker-v5drmq annotated
+```
+
+ 
+**Step 3**
+
+Check the MachineDeployment description
+
+```bash
+$ kubectl describe machinedeployments -n kube-system test-cluster-worker-v5drmq
+
+Name:         test-cluster-worker-v5drmq
+
+Namespace:    kube-system
+
+Labels:       &lt;none>
+
+Annotations:  cluster.k8s.io/cluster-api-autoscaler-node-group-max-size: 5
+
+              cluster.k8s.io/cluster-api-autoscaler-node-group-min-size: 1
+
+              machinedeployment.clusters.k8s.io/revision: 1
+
+API Version:  cluster.k8s.io/v1alpha1
+
+Kind:         MachineDeployment
+
+Metadata:
+
+  Creation Timestamp:  2021-07-23T11:05:11Z
+
+  Finalizers:
+
+    foregroundDeletion
+
+  Generate Name:  test-cluster-worker-
```

Comment:
Is this correct or the one from the `/master/` file?

Reply:
Modified